codec-registration-checker-compiler-plugin/src/test/scala/org/virtuslab/ash/CodecRegistrationCheckerCompilerPluginSpec.scala

```diff
@@ -26,7 +28,7 @@ class CodecRegistrationCheckerCompilerPluginSpec extends AnyWordSpecLike with sh
       File.usingTemporaryDirectory() { directory =>
         val out = CodecRegistrationCheckerCompiler.compileCode(
           serializersCode(0) :: dataSourceCode,
-          List(s"${directory.toJava.getAbsolutePath}"))
+          List(s"${directory.toJava.getAbsolutePath}", s"--source-code-directory=$sourceCodeDirectory"))
```

Comment:
Nit: pls add a case where `--source-code-directory` is **not** passed and exception is thrown

Reply:
sure, test added